website/blog/2024-10-07-release-1.13.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Podman Desktop 1.13 Release
+description: Podman Desktop 1.13 has been released!
+slug: podman-desktop-release-1.13
+authors: SoniaSandler
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /blog/img/podman-desktop-release-1.13/banner.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.13 Release! 🎉
+
+![podman-desktop-hero-1.13](img/podman-desktop-release-1.13/banner.png)
+
+Podman Desktop 1.13 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **HyperV support**: You can now create and manage HyperV Podman machines directly from Podman Desktop.
+- **Search image feature**: search for an image from Podman Desktop.
+- **Updated empty state pages**: start your journey in the containers, images, pods, and Kubernetes pages with a click of a button
+- **New Kubernetes navigation**: All Kuberenetes related pages have been moved to a separate sub menu for easier and less crowded navigation.
+
+<!--truncate-->
+
+## Release Details
+
+### HyperV support
+
+Podman Desktop is now supporting the creation and modification of HyperV based Podman machines in Windows directly from the app.
+When creating a podman machine in Windows, if both HyperV and WSL are enabled, you can choose either one of them as the provider from the dropdown menu.
+
+### Search image feature
+
+Want to pull an image but forgot its full name or tag? Now when pulling an image, a list of possible images is shown based on the provided input to make it easier for your to find and pull the right image. Just enter an image name and look for the right one in the list. If you would like to search for a specific tag, just add : to the name.
```

Comment:
`Now when pulling an image, a list of possible images is shown based on the provided input to make it easier for your to find and pull the right image. Just enter an image name and look for the right one in the list.` is a bit of a mouthful.
How about just `As you type a list of likely images is now shown to help you find the right one and reduce typing` along with a small image?